### PR TITLE
Refactor connectDB method

### PR DESCRIPTION
**Pull request by Hacksore**

### CODE REVIEW

**Comment by Hacksore** [comment]
@Technoash please try this implementation and let me know if that works.

**Comment by Hacksore** [comment]
It is working for me:
image

**Comment by Technoash** [comment]
You're right. That works in a much cleaner way.

I used a non promise MySQL package so the promise was initially required. When I swapped it out for the async-mysql package I didn't refactor.

Thanks!

File: connectdb.js [hunk 1]
```diff
@@ -3,25 +3,26 @@ let mysql = require('async-mysql')
...
             password : 'animal',
             database : 'world'
-        }).then((con)=>{
-            connection = con;
-            console.log("connected")
-            res(connection)
         })
-    })
+    } catch (err) {
+        return null
```
Comment (Technoash) [inline connectdb.js:21]: Probably should let connectDB error out instead of returning null

**Pull request merged**